apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -370,6 +397,12 @@ def _get_nrf_device_args(options):
         type="bool",
         help="Run on the FVP emulator instead of hardware.",
     ),
+    server.ProjectOption(
+        "heap_size",
```

Comment:
nit: heap_size_bytes

Reply:
should HEAP_SIZE and DEFAULT_HEAP_SIZE also have _BYTES suffix?

Reply:
added the _BYTES suffix to all three.